src/app.ts

```diff
@@ -40,8 +45,14 @@ namespace AppStatus {
     }
 }
 
+/**
+ * Export the AppStatus enum
+ */
 export { AppStatus };
 
+/**
+ * Interface for App data
```

Comment:
please define properties of the interface. See Typedef here https://devhints.io/jsdoc

Reply:
also add a description of what this is used for.